### PR TITLE
Add unit test for JobService Logger

### PR DESCRIPTION
**Pull request by mmpei**
add unit test for JobService Logger.

Signed-off-by: peimingming <peimingming@corp.netease.com>

### CODE REVIEW

**Comment by mmpei** [comment]
@steven-zou  add UT as we talked, please review.

File: src/jobservice/logger/factory.go [hunk 1]
```diff
@@ -81,5 +81,9 @@ func DBFactory(options ...OptionItem) (Interface, error) {
 		}
 	}
 
+	if len(key) == 0 {
```
Comment (ywk253100) [inline src/jobservice/logger/factory.go]: Is there any default value for `key` if user doesn't set it in `config.yaml`?

Comment (mmpei) [reply src/jobservice/logger/factory.go]: this is a Job UUID, not config parameters. and this is the unique index for DB log data, must exists.

Comment (ywk253100) [reply src/jobservice/logger/factory.go]: thanks

**Pull request merged**